src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1954,34 +1986,40 @@ private void VisitArgumentEvaluate(ImmutableArray<BoundExpression> arguments, Im
             }
         }
 
-        private static ImmutableArray<BoundExpression> RemoveArgumentConversions(ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> refKindsOpt)
+        private static (ImmutableArray<BoundExpression>, ImmutableArray<Conversion>) RemoveArgumentConversions(ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> refKindsOpt)
         {
             int n = arguments.Length;
+            var conversions = default(ImmutableArray<Conversion>);
             if (n > 0)
             {
                 var argumentsBuilder = ArrayBuilder<BoundExpression>.GetInstance(n);
+                var conversionsBuilder = ArrayBuilder<Conversion>.GetInstance(n);
                 bool includedConversion = false;
                 for (int i = 0; i < n; i++)
                 {
                     RefKind refKind = GetRefKind(refKindsOpt, i);
                     var argument = arguments[i];
+                    var conversion = Conversion.Identity;
                     if (refKind == RefKind.None)
```

Comment:
How about `RefKind.In`? Should it behave like `RefKind.None`?

Reply:
Added `// PROTOTYPE` comment.

---
In reply to: [189377421](https://github.com/dotnet/roslyn/pull/26356#discussion_r189377421) [](ancestors = 189377421)